homeassistant/components/bmw_connected_drive/coordinator.py

```diff
@@ -31,6 +31,8 @@ class BMWDataUpdateCoordinator(DataUpdateCoordinator[None]):
     """Class to manage fetching BMW data."""
 
     account: MyBMWAccount
+    read_only: bool = False
```

Comment:
You're setting a default which isn't really in line with the title of the PR.
Also it seems it's mandatory coming from config entry options so I think we can just leave it typing and remove the default `False`

Reply:
Makes perfect sense, thank you!